frontend/router/index.tsx

```diff
@@ -92,6 +91,7 @@ const routes = (
               <IndexRedirect to={"dashboard"} />
               <Route path="dashboard" component={Homepage} />
               <Route path="settings" component={AuthAnyAdminRoutes}>
+                <IndexRedirect to={"teams"} />
```

Comment:
Only fleet premium has access to teams, so I think this should redirect to "organization" instead

Reply:
Good point. I was thinking everyone could access `/teams`, but you're right, not free users.

I've updated to redirect to `/dashboard` instead. Ideally we could check `isOnGlobalTeam` within the router to determine if they should be redirected to `/organizations` or `/teams`, but that context does not seem to be available inside the router. 

I think redirecting to `/dashboard` is okay in this case, since it's an edge case of the user manually editing their URL, so I don't think many will run into it. We just want to stop it from loading a blank page.